src/core/display/DisplayObject.js

```diff
@@ -431,6 +431,25 @@ DisplayObject.prototype.generateTexture = function (renderer, scaleMode, resolut
 };
 
 /**
+ * Set the parent Container of this DisplayObject
+ *
+ * @param container {Container} The Container to add this DisplayObject to
+ * @return {DisplayObject} This DisplayObject
+ */
+DisplayObject.prototype.setParent = function (container)
+{
+    if (!container || !container.addChild)
+    {
+        throw new Error('setParent: Argument must be a Container');
+    }
+    else
+    {
+        container.addChild(this);
+        return this;
```

Comment:
`addChild` returns the child, you can just do `return container.addChild(this)`.